### PR TITLE
[#OSF-58348]Email help desk with stuck registrations

### PR DESCRIPTION
**Pull request by chennan47**
## Purpose

Change the scripts which handle failed registration not to cleanup the failed one but to collect the failed registration daily and send all the info to help desk. Replace of https://github.com/CenterForOpenScience/osf.io/pull/5480.

## Changes

1.remove the code in https://github.com/CenterForOpenScience/osf.io/blob/6ccd9126c397556b1117682285cbfe862d718824/scripts/cleanup_failed_registrations.py which cleanup the failed registration

2.log all the failed registration and send it to the help desk

3.set it up as a daily task through celerybeat

## Ticket

https://openscience.atlassian.net/browse/OSF-5848

### CODE REVIEW

File: scripts/get_failed_registrations.py [hunk 1]
```diff
@@ -0,0 +1,82 @@
+
+# -*- coding: utf-8 -*-
+import sys
+import logging
+
+from datetime import datetime
+from django.core.mail import send_mail
```
Comment (sloria) [inline scripts/get_failed_registrations.py]: Use the OSF's `send_mail` function rather than Django's. Otherwise, this script will likely crash.

**Comment by sloria** [comment]
Please test that the script works, and that the correct email output is logged to the celery logs.

**Comment by sloria** [comment]
Pass finished.

File: scripts/get_failed_registrations.py (hunk 1 above)
Comment (chennan47) [inline scripts/get_failed_registrations.py]: fixed in fa197cc

**Pull request closed without merging**